pkg/controller/scansettingbinding/scansettingbinding_controller.go

```diff
@@ -183,27 +182,11 @@ func (r *ReconcileScanSettingBinding) Reconcile(ctx context.Context, request rec
 			}
 		}
 
-		scan, product, err := newCompScanFromBindingProfile(r, instance, profileObj, log)
+		scan, _, err := newCompScanFromBindingProfile(r, instance, profileObj, log)
```

Comment:
Looks like this is the only place where this is called, and since we're not using `product` anymore we can probably clean it up in another patch.